src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -73,17 +75,36 @@ public void mecanumDrive(double joystickX, double joystickY, double rotation) {
 		if (Math.abs(backRightPower) > ControlConstants.MAX_ROBOT_SPEED)
 			backRightPower *= ControlConstants.MAX_ROBOT_SPEED / Math.abs(backRightPower);
 
+		//finetuned driving system
+		if (finetuned = true) {
+				frontRightPower = frontRightPower/3;
+				frontLeftPower = frontLeftPower/3;
+				backRightPower = backRightPower/3;
+				backLeftPower= backLeftPower/3;
+
```

Comment:
this is great!
As `finetuned` is a `private` variable, you should create a Command that changes the value, such that we can bind a controller button to this command and update the value from outside of the subsystem on the press of a button. This is more efficient and cleaner than constantly polling the value every roborio cycle. See https://github.com/Rundle-Robotics/2023-Charged-Up/blob/example/src/main/java/frc/robot/subsystems/ExampleSubsystem.java#L19